### PR TITLE
replace glide with dep

### PR DESCRIPTION
**Pull request by weirdgiraffe**
use state of the art dependency management tool

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ go:
   - 1.9
 
 install:
-  - go get github.com/Masterminds/glide
+  - go get github.com/golang/dep/cmd/dep
```
Comment (aeneasr) [inline .travis.yml]: Can we use the binaries here instead? Otherwise we might use something unstable in the tests - plus it reduces build time.

**Comment by aeneasr** [comment]
Thank you! Just one thing :)

File: .travis.yml (hunk 1 above)
Comment (weirdgiraffe) [reply .travis.yml]: sure, will fix that

**Comment by weirdgiraffe** [comment]
I've replaced `go get` with binary download for `dep` and squashed this commit into PR to keep history clean.

**Comment by aeneasr** [comment]
Thank you!

**Pull request merged**